ups/spectractor.table

```diff
@@ -0,0 +1,13 @@
+# For each dependency except python, list it here along with its minimum
+# version number.
+# Very common third-party packages (boost, python, swig, doxygen) 
+# and very low-level LSST packages can be assumed to be recursively
+# included by low-level LSST packages such as utils or daf_base.
+# Any other package whose interface is used should be listed explicitly
+# rather than assuming it will be included recursively.
+
+# The following is boilerplate for all packages.
+# See Tech Note DMTN-001 for details on LSST_LIBRARY_PATH
+# Pure-Python packages only need the last line.
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
+envPrepend(PATH, ${PRODUCT_DIR}/bin)
```

Comment:
If there's no `bin` or `bin.src` directory, I'm not sure why anything is being added to `PATH`.

Reply:
Yeah, just copying and pasting from what I did on `atmospec`, definitely unnecessary, thanks.